src/Uno.UI.Runtime.Skia.WebAssembly.Browser/Uno.UI.Runtime.Skia.WebAssembly.Browser.csproj

```diff
@@ -53,6 +54,14 @@
 			<Pack>true</Pack>
 			<PackagePath>build</PackagePath>
 		</Content>
+		<Content Include="buildTransitive/*.*">
+			<Pack>true</Pack>
+			<PackagePath>buildTransitive</PackagePath>
+		</Content>
+		<Content Include="icu/unoicu/unoicu.a">
+			<Pack>true</Pack>
+			<PackagePath>buildTransitive</PackagePath>
+		</Content>
```

Comment:
This is likely going to duplicate the assets, some coming from uno.icu-wasm, and others from the runtime.browser package. In the final app, those assets should be integrated automatically by the buildTransitive that comes from uno.isu-wasm. (same for the other targets)